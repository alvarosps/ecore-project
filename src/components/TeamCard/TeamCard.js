import React from 'react';
import PropTypes from 'prop-types';

import { Card } from './TeamCard.styles';

const TeamCard = ({ teamName }) => {
    return (
        <Card>
            {teamName}
        </Card>
    )
}

TeamCard.propTypes = {
    teamName: PropTypes.string.isRequired
}

export default TeamCard;