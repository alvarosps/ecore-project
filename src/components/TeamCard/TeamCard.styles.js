import styled from 'styled-components';

export const Card = styled.div`
    transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
    border-radius: 4px;
    box-shadow: rgba(0, 0, 0, 0.2) 0px 2px 1px -1px, rgba(0, 0, 0, 0.14) 0px 1px 1px 0px, rgba(0, 0, 0, 0.12) 0px 1px 3px 0px;
    background-image: linear-gradient(rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0.05));
    background-color: rgb(26, 32, 39);
    font-family: "Roboto", "Helvetica", "Arial", sans-serif;
    font-weight: 400;
    font-size: 0.875rem;
    line-height: 1.43;
    letter-spacing: 0.01071em;
    padding: 16px;
    text-align: center;
    color: rgba(255, 255, 255, 0.7);

    :hover {
        background-color: rgb(73, 80, 89);
    }
`;