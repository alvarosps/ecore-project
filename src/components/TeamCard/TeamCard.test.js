import React from 'react';
import renderer from 'react-test-renderer';
import TeamCard from './TeamCard';

describe("TeamCard", () => {
    it('Should render a TeamCard', () => {
        const teamCard = renderer.create(<TeamCard teamName='test' />);
        
        expect(teamCard.toJSON()).toMatchSnapshot()
        expect(teamCard.toJSON().children[0]).toEqual('test');
    })
});