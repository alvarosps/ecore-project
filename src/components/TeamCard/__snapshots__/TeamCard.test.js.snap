// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamCard Should render a TeamCard 1`] = `
<div
  className="sc-bczRLJ hlyZLj"
>
  test
</div>
`;
